src/api/functions.js

```diff
@@ -1,4 +1,28 @@
 import APIFactory from './factory.js';
 import credentials from './credentials.js';
 
-export default APIFactory({ credentials, fetch });
\ No newline at end of file
+// import newReleases from './releases.json';
+// import album from './album.json';
+
+// const fetch = (url) => Promise.resolve(
+//   {
+//     json: () => {
+//       let result = [];
+//       if (url.match('^https://api.spotify.com/v1/albums/')) {
+//         result = album;
+//       } else if (url.includes('new-releases')) {
+//         result = newReleases;
+//       }
+//       return Promise.resolve(result);
+//     },
+
+//     text: () => Promise.resolve(JSON.stringify([
+//       {
+//         item_id: '4wKGNANAC9OHjccAiihjGO',
+//         likes: 5,
+//       },
+//     ])),
+//   },
+// );
```

Comment:
- [ ] It will be nice to remove all unused code in order to have a clean codebase. It's standard practice.

Reply:
Thanks, 👍